.gitignore

```diff
@@ -0,0 +1,7 @@
+CMakeCache.txt
+CMakeFiles/
+Makefile
+cmake_install.cmake
+examples/CMakeFiles/
+examples/Makefile
+examples/cmake_install.cmake
```

Comment:
This is useful, I also wanted to add it, when playing with the repo it's kinda boring to deal with untracked stuff.